### PR TITLE
Add handling in upseto.pth file.

### PR DESCRIPTION
**Pull request by yogev-stratoscale**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,5 @@
 build
 dist
 upseto.egg-info
-.coverage
+.coverage*
```
Comment (eyal-stratoscale) [inline .gitignore]: why coverage?

File: setup.py [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
 from setuptools import setup
 
-SITE_PACKAGES = '/usr/lib/python2.7/site-packages'
-DIST_PACKAGES = '/usr/lib/python2.7/dist-packages'
+from distutils import sysconfig
+site_packages_path = sysconfig.get_python_lib()
 
-
-data_files = []
+data_files = [(site_packages_path, ["conf/upseto.pth"])]
```
Comment (eyal-stratoscale) [inline setup.py]: this also should be only if userid==0

File: .gitignore (hunk 1 above)
Comment (yogev-stratoscale) [inline .gitignore]: This existed before my time shlomp had coverage but some how i got new files that showed in git.

File: setup.py (hunk 2 above)
Comment (yogev-stratoscale) [inline setup.py]: This will be taken care of by get_python_lib() function --- on venv it will eval correctly.

**Pull request merged**